knip.jsonc

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://unpkg.com/knip@3/schema.json",
+    "entry": [
+        "Herebyfile.mjs",
+        // Knip does not understand Herebyfile.mjs (a new Knip plugin could probably handle it), so we add the entry files manually:
+        "src/cancellationToken/cancellationToken.ts",
+        "src/testRunner/_namespaces/Harness.ts",
+        "src/tsc/tsc.ts",
+        "src/tsserver/server.ts",
+        "src/typescript/typescript.ts",
+        "src/typingsInstaller/nodeTypingsInstaller.ts",
+        "src/watchGuard/watchGuard.ts",
+
+        // src/testRunner/tests.ts does not export anything, so we add all test files separately instead:
+        "src/testRunner/unittests/**/*.ts",
```

Comment:
I don't quite understand this; why doesn't `tests.ts` work? What differs if that file is used instead?

Reply:
Me neither, seems to work fine now :)